css/style.css

```diff
@@ -1,3 +1,47 @@
 h1 {
   color: salmon;
 }
+body {
+  text-align: center;
+  margin: 0 auto;
+  max-width: 700px;
+  background: #EDEFEC;
+}
+
+.imagebox1, .imagebox2 {
+  display: flex;
+  justify-content: space-around;
+}
+
+img {
```

Comment:
We have to be careful with broad selectors like this. In the assignment we only had those 6 images to worry about, but if this were for a larger site with`<img>` tags that needed multiple sizes on several pages, it'd be hard to manage.

Consider bundling these properties into a new css class, or just be more specific with your selector:
`.box img` for instance only changes the images contained inside `.box` elements, instead of every `img` on the page.